src/app/routes/settings/index.tsx

```diff
@@ -1,9 +1,18 @@
-import { type FC } from 'react';
+import { json } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
 
 import { SettingsPage } from '@/screens/settings';
 
-const Page: FC = () => {
-  return <SettingsPage />;
+export const loader = () => {
+  return json({
+    version: process.env.PUBLIC_APP_VERSION,
```

Comment:
would it be taken in runtime?
I would prefer set it only in build time

Reply:
You could use local `.env` with `PUBLIC_APP_VERSION=$npm_package_version` in order to see app's version during development

Reply:
that's ok, but I'm worried that at the moment of program execution this variable will no longer exist.

Reply:
Why? We've set it during build and passed into Docker container
```dockerfile
ARG PUBLIC_APP_VERSION
ENV PUBLIC_APP_VERSION=${PUBLIC_APP_VERSION}
```
Otherwise I don't clearly see what you mean 🤷‍♂️